.travis.yml

```diff
@@ -29,6 +31,8 @@ install:
         pip install -U tox ;;
       docs)
         pip install -U sphinx ;;
+      quality)
+        pip install -rrequirements/quality.txt
```

Comment:
Well, we could specify different `testenv` for `docs` and `quality`.

But I would rather do it in separate ticket.